.vscode/tasks.json

```diff
@@ -0,0 +1,14 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "npm",
+			"script": "lint",
+			"problemMatcher": [
+				"$eslint-stylish"
+			],
+			"label": "npm: lint",
+			"detail": "eslint --ext .js,.vue --ignore-path .gitignore ."
+		}
```

Comment:
This will let us do `> Run lint` from VSCode and show errors from all files in the problems pane which is really nice.